packages/edit-site/src/components/page-patterns/dataviews-patterns.js

```diff
@@ -235,12 +253,23 @@ export default function DataviewsPatterns() {
 				render: ( { item } ) => (
 					<Title item={ item } categoryId={ categoryId } />
 				),
-				maxWidth: 400,
 				enableHiding: false,
 			},
-		],
-		[ view.type, categoryId ]
-	);
+		];
+		if ( type === PATTERN_TYPES.theme ) {
+			_fields.push( {
+				header: __( 'Sync Status' ),
+				id: 'sync-status',
+				isVirtual: true,
```

Comment:
Why should this be a virtual filter. For me it's actually a valid field that can show Sync or not.

Reply:
We could definitely delay this `virtual` concept for now, and by not providing a `render` prop and adjusting the views to check for that, we could not display the field. My exploration was based on thoughts I had for other cases too, as I describe in the PR's description.

Reply:
To be honest, I can't think of a case where a filter doesn't make sense as a regular field personally.